### PR TITLE
docs: add requirements around strong time synchronisation

### LINKED ISSUE #91: Add a note on criticality of time synchronisation
Tokens have short validity periods so it is important that auth servers and clients are well-synchronised, likely via the availability of NTP in the deployment network.

### PR DESCRIPTION
**Pull request by andrewbonney**
Resolves #91

### CODE REVIEW

File: docs/4.0. Behaviour.md [hunk 1]
```diff
@@ -9,3 +9,15 @@ IS-10 defines behaviour for:
 -   [Token Requests](4.3.%20Behaviour%20-%20Token%20Requests.md)
 -   [Access Tokens](4.4.%20Behaviour%20-%20Access%20Tokens.md)
 -   [Resource Servers](4.5.%20Behaviour%20-%20Resource%20Servers.md)
+
+
+## Time Synchronisation
```
Comment (peterbrightwell) [inline docs/4.0. Behaviour.md]:
```suggestion
## Time Synchronization
```

File: docs/4.0. Behaviour.md [hunk 2]
```diff
@@ -9,3 +9,15 @@ IS-10 defines behaviour for:
...
+
+This specification is dependent upon accurate time synchronisation between all system components.
```
Comment (peterbrightwell) [inline docs/4.0. Behaviour.md]:
```suggestion
This specification is dependent upon accurate time synchronization between all system components.
```

File: docs/4.0. Behaviour.md [hunk 3]
```diff
@@ -9,3 +9,15 @@ IS-10 defines behaviour for:
...
+
+Authorization Servers, Resource Servers and Clients (or the hosts which they are deployed onto) MUST be capable of
+synchronising their clocks to an external source of time. It is STRONGLY RECOMMENDED that Network Time Protocol (NTP) is
```
Comment (peterbrightwell) [inline docs/4.0. Behaviour.md]:
```suggestion
synchronizing their clocks to an external source of time. It is STRONGLY RECOMMENDED that Network Time Protocol (NTP) is
```

File: docs/4.0. Behaviour.md [hunk 4]
```diff
@@ -9,3 +9,15 @@ IS-10 defines behaviour for:
...
+supported for this purpose.
+
+Deployment environments SHOULD include one or more providers of time synchronisation data matching the requirements of
```
Comment (peterbrightwell) [inline docs/4.0. Behaviour.md]:
```suggestion
Deployment environments SHOULD include one or more providers of time synchronization data matching the requirements of
```

**Review by peterbrightwell — APPROVED** [review]
Looks good (with spelling change to be consistent with "authorization")

**Comment by garethsb** [comment]
I was going to say that consistent zedification was just as important as consistent timing, but I guess I mean consistent zeeification :-/

**Pull request merged**